Application/SchedulingLib/SchedulingContext.cs

```diff
@@ -8,7 +8,12 @@ namespace SchedulingLib
 {
     class SchedulingContext : DbContext
     {
-        public DbSet<Instructor> Instructors { get; set; }
+        public DbSet<Building> Building { get; set; }
+        public DbSet<Room> Room { get; set; }
+        public DbSet<Section> Section { get; set; }
+        public DbSet<Course> Course { get; set; }
+        public DbSet<Instructor> Instructor { get; set; }
```

Comment:
These can be plural... these DbSet objects are how we create/update/delete each model from the database, so it makes sense to say something like `Buildings.Add(...)`